keyboards/shuguet/shu89/keymaps/default/keymap.json

```diff
@@ -0,0 +1,28 @@
+{
+    "keyboard": "shuguet/shu89",
+    "keymap": "default",
+    "config": {
+        "features": {
+            "via": true
+        }
+    },
```

Comment:
```suggestion
```

Reply:
Is there some issue with the VIA project I am not aware of?
This snippet was recommended to me on Discord to get VIA working, and seeing as the `.gitignore` of the repo purposefully ignore the `via` subfolder in every keyboard's `keymaps` folder, how am I supposed to enable VIA support in a purely data-driven/JSON-based keyboard & keymap config?

Reply:
Got it, thank you, I missed that.
Removed from the PR.